main_test.go

```diff
@@ -2,26 +2,35 @@ package main
 
 import (
 	"flag"
+	"io"
+	"log"
 	"os"
 	"testing"
 
-	componenttest "github.com/ONSdigital/dp-component-test"
+	"github.com/ONSdigital/dp-import-cantabular-dataset/config"
 	"github.com/ONSdigital/dp-import-cantabular-dataset/features/steps"
 	"github.com/cucumber/godog"
 	"github.com/cucumber/godog/colors"
 )
 
-var componentFlag = flag.Bool("component", false, "perform component tests")
+var (
+	componentFlag = flag.Bool("component", false, "perform component tests")
 
-type ComponentTest struct {
-	MongoFeature *componenttest.MongoFeature
-}
+	logFileName = "log-output.txt"
+)
+
+type ComponentTest struct{}
 
 func (f *ComponentTest) InitializeScenario(ctx *godog.ScenarioContext) {
-	component := steps.NewComponent()
+	component, err := steps.NewComponent()
+	if err != nil {
+		log.Panicf("unable to initialise component: %s", err)
```

Comment:
I think it's best not to panic : ) - that's what I've been told anyway
```suggestion
		os.Exit(1)
```

Reply:
There are times when panicking is better than a raw exit.. although here it's basically of no consequence either way :) When calling os.Exit() no deferred functions are called, which can be an issue. Panicking is usually frowned upon when it's done at random points through the code but that at the very top level it's usually fine. I'm happy to change it here to keep in line with code elsewhere though as it really doesn't matter either way